packages/api/src/api/project/id/get.ts

```diff
@@ -0,0 +1,21 @@
+import { Project } from '@hangar/database';
+import { Request, Response } from 'express';
+import { logger } from '../../../utils/logger';
+
+export const get = async (req: Request, res: Response) => {
+  const {
+    entityManager,
+    params: { id: projectId },
+  } = req;
+  try {
+    const project = await entityManager.find(Project, { id: projectId as string });
+    if (!project) {
+      res.sendStatus(404);
```

Comment:
Missing a return after this; it'll fall through and you'll get an error like `cannot set headers after they have been set`